### PR TITLE
feat(ME-2736): extended policy actions

### PR DESCRIPTION
**Pull request by btoonk**
# Description

extended policy actions
Fixes # (ME-2736)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested using localdev with old and new policie.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: cmd/client/ssh/ssh.go [hunk 1]
```diff
@@ -89,7 +91,7 @@ var sshCmd = &cobra.Command{
 		}
 		socketPref := preference.NewSSHSocket(hostname)
 
-		if sshType != "aws-ssm" {
+		if !hasUpstreamUsername && sshType != "aws-ssm" {
```
Comment (adrianosela) [inline cmd/client/ssh/ssh.go:94]: if this bool is only used here, prefer to use pickedHost.HasUpstreamUsername directly.

File: internal/api/models/policy.go [hunk 2]
```diff
@@ -79,3 +81,71 @@ type PolicyActionUpdateRequest struct {
...
+	MaxSessionDurationSeconds int                    `json:"max_session_duration_seconds" mapstructure:"max_session_duration_seconds"`
+}
+
+type DatabaseSchemaAction struct {
+	Schema            string   `json:"schema" mapstructure:"schema"`
+	AllowedQueryTypes []string `json:"allowed_query_types" mapstructure:"allowed_query_types"`
+	ReadOnly          bool     `json:"read_only" mapstructure:"read_only"`
+}
+
+type SSHActions struct {
```
Comment (adrianosela) [inline internal/api/models/policy.go:103]: *sigh*... we should just make the policy lib part of the public SDK.... There's no real magic there.

**Review by adrianosela — CHANGES_REQUESTED** [review]
Can we split the added exec functionality from the policy changes?

File: cmd/client/ssh/ssh.go (hunk 1 above)
Comment (btoonk) [reply cmd/client/ssh/ssh.go:94]: we only have pickedHost if the hostname is empty. (it's inside an if statement)

**Pull request merged**